content/en/blog/_posts/2020-12-04-gsod-2020-improving-api-reference-experience.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: blog
+title: "GSoD 2020: Improving the API Reference Experience"
+date: 2020-12-04
+slug: gsod-2020-improving-api-reference-experience
+---
+
+**Author**: [Philippe Martin](https://github.com/feloy)
+
+# Introduction
+
+[Google Season of Docs](https://developers.google.com/season-of-docs) brings open source organizations and technical writers together, to spend three months working closely during the autumn on a specific documentation project.
+
```

Comment:
nit: You could leave out "during the autumn". I think GSOD runs at different times of year.